### ISSUE TITLE
<script> tag bug

### ISSUE THREAD

**Issue by Kelbazz** [atom/atom#21804]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

problem with the coloration / auto-completion when I create a script tag with help of auto-completion.

### Steps to Reproduce

1. Create HTML file.
2. Type script and use auto-completion.

**Expected behavior:**

That the coloration work normally, like other tags.

**Actual behavior:**

The coloration stop to work and all the tag become white.

**Reproduces how often:**

Always (except in safe mode).

#### Atom :

Atom       : 1.55.0
Electron  : 6.1.12
Chrome  : 76.0.3809.146
Node      : 12.4.0

#### Atom Package Manager :

apm : 2.5.2
npm : 6.14.8
node : 12.4.0 x64
atom : 1.55.0
python 
git : 2.20.1

#### Operating system

type : Chrome OS (Crostini)
version : Debian GNU/Linux 10 (buster) on Chrome OS 89.0.4389.116 (Build officiel) (64 bits)

### Additional Information

I haven't installed package, but in the safe mode of atom this work correctly.

#### Screenshots

![Screenshot 2021-01-16 at 16 38 40](https://user-images.githubusercontent.com/49045228/104816190-b7711400-5819-11eb-9cd7-761347c1d8b9.png)
![Screenshot 2021-01-16 at 16 38 11](https://user-images.githubusercontent.com/49045228/104816191-b8a24100-5819-11eb-82e4-576cb07bf3bf.png)

**Comment by sadick254** [comment]
Could you provide screenshots to help us debug what the issue could be.

**Comment by Kelbazz** [comment]
ok @sadick254

**Comment by Kelbazz** [comment]
Here are the screenshots !

**Comment by ThatXliner** [comment]
Have you tried turning off tree sisters? It's probably a problem with tree sisters

**Comment by ggstuart** [comment]
I have this exact issue, I don't know what tree sisters is.

**Comment by smilers** [comment]
dankjewel voor de klare taal.. ik ben een baby in git

**Comment by ThatXliner** [comment]
> I have this exact issue, I don't know what tree sisters is.

TL;DR: Tree sisters is a new parsing system for Atom, as opposed to TextMate-style regex grammars (tree-sisters is fast). It may be buggy for some languages though.

[The blog](https://github.blog/2018-10-31-atoms-new-parsing-system/) and [the homepage](https://tree-sitter.github.io/tree-sitter/). The repo's [here](https://github.com/tree-sitter/tree-sitter).

**Comment by Kelbazz** [comment]
Mmm... I don't know why but the bug still working in 1.55.0 !

**Comment by ThatXliner** [comment]
Try Atom beta. A lot of changed since the writing of that comment and #21713 might have fixed it

**Comment by ThatXliner** [comment]
Try now

**Comment by Kelbazz** [comment]
Thx @ThatXliner !
